project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Order/preview.html.twig

```diff
@@ -20,7 +20,7 @@
                         {{ 'Promo code'|trans }} {{ (-orderPreview.promoCodeDiscountPercent)|formatPercent }}
                     </td>
                     <td class="table-cart-preview__cell table-cart-preview__cell--price">
-                        {{ (-quantifiedProductDiscount.priceWithVat)|price }}
+                        {{ (-quantifiedProductDiscount.priceWithVat.toValue)|price }}
```

Comment:
```suggestion
                        {{ quantifiedProductDiscount.priceWithVat.multiply('-1')|price }}
```

I don't know if it works, but it would be more Money-friendly; also right now we are using deprecated behavior

Also I would like to have `negative` method instead of `multiply(-1)` as mentioned earlier :-)

Reply:
I've implemented `inverse` to handle this in 6a9b1dbd3